app/schedulers/complete_mentorship_cron_job.py

```diff
@@ -12,15 +12,16 @@ def complete_overdue_mentorship_relations_job():
     with application.app_context():
         from app.utils.enum_utils import MentorshipRelationState
         from app.database.models.mentorship_relation import MentorshipRelationModel
-        all_relations = MentorshipRelationModel.query.all()
+        all_relations = filter(
+            lambda relation: relation.state == MentorshipRelationState.ACCEPTED
+                             and relation.end_date < current_date_timestamp,
+            MentorshipRelationModel.query.all())
 
         current_date_timestamp = datetime.now().timestamp()
 
         for relation in all_relations:
+            relation.state = MentorshipRelationState.COMPLETED
+            relation.save_to_db()
 
-            if relation.state == MentorshipRelationState.ACCEPTED and relation.end_date < current_date_timestamp:
-                relation.state = MentorshipRelationState.COMPLETED
-                relation.save_to_db()
-
-            # for tests purposes
-            # print('{} | hola hola hola hola hola'.format(datetime.now()))
+        # for tests purposes
+        # print('{} | hola hola hola hola hola'.format(datetime.now()))
```

Comment:
Maybe we forgot to omit this.

Reply:
I'm sorry. I didn't get your point. Do you mean that we forgot to delete this comment?

Reply:
I think this part git autoformatted and therefore added to the commit.

Reply:
Thanks for the quick response! :) Now I understand what you mean. I think this should be changed, as the if statement was deleted and it means that there is no need for one more tab (4 spaces) at the beginning.